### PR TITLE
Simplify classification custom loop example

### PR DESCRIPTION
**Pull request by okuta**
rel #3832

### CODE REVIEW

File: examples/mnist/train_mnist_custom_loop.py [hunk 1]
```diff
@@ -88,18 +86,14 @@ def main():
...
-                        for batch in test_iter:
-                            x, t = convert.concat_examples(batch,
-                                                           args.gpu)
-                            x = chainer.Variable(x)
-                            t = chainer.Variable(t)
-                            loss = model(x, t)
-                            sum_loss += float(loss.data) * len(t.data)
-                            sum_accuracy += (float(model.accuracy.data) *
-                                             len(t.data))
+                # This is optional but can reduce computational overhead.
```
Comment (kmaehashi) [inline examples/mnist/train_mnist_custom_loop.py]: I think it's better to say that only `enable_backprop` is optional. (setting `train` to `False` is not optional)

Comment (okuta) [reply examples/mnist/train_mnist_custom_loop.py]: Thanks! I will fix

**Comment by okuta** [comment]
I fixed.

**Comment by kmaehashi** [comment]
LGTM!

**Pull request merged**